api/capi/include/nnstreamer-single.h

```diff
@@ -185,6 +185,33 @@ int ml_single_set_input_info (ml_single_h single, const ml_tensors_info_h info);
  */
 int ml_single_set_timeout (ml_single_h single, unsigned int timeout);
 
+/**
+ * @brief Sets the property value for the given model.
+ * @details Note that a model/framework may not support to change the property after opening the model.
+ * @since_tizen 6.0
+ * @param[in] single The model handle.
+ * @param[in] name The property name.
+ * @param[in] value The property value.
+ * @return @c 0 on success. Otherwise a negative error value.
+ * @retval #ML_ERROR_NONE Successful
+ * @retval #ML_ERROR_NOT_SUPPORTED Not supported.
+ * @retval #ML_ERROR_INVALID_PARAMETER Fail. The parameter is invalid.
+ */
+int ml_single_set_property (ml_single_h single, const char *name, const char *value);
+
+/**
+ * @brief Gets the property value for the given model.
```

Comment:
How about adding a note that the behavior of this function changes before and after an invoke?
Before the first invoke call, the properties can be updated freely as the framework is not yet open.
However, after the first invoke (`ml_single_invoke`), the support of updating the properties will be limited depending on the support provided by the framework.